hack/collector-default

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# -------------------------------------------------------------------------------- #
```

Comment:
Suggest using `#` instead for consistency. Ref: https://github.com/rancher/support-bundle-kit/blob/master/hack/collector-harvester#L67-L69

Reply:
Got it, I've replaced the equal sign with a pound sign.